tests/cucumber/features/features.given.js

```diff
@@ -0,0 +1,14 @@
+import { browser } from '@wdio/globals'
+import { Given } from '../../../packages/wdio-cucumber-framework/build/index.js'
```

Comment:
we use pnpm so we can just reference this as normal package
```suggestion
import { Given } from '@wdio/cucumber-framework'
```